recipes-devtools/upm/upm_0.7.1.bb

```diff
@@ -16,16 +16,14 @@ inherit distutils-base pkgconfig python-dir cmake
 CFLAGS_append_edison = " -msse3 -mfpmath=sse"
 
 FILES_${PN}-doc += " ${datadir}/upm/examples/"
-RDEPENDS_${PN} += " mraa"
+RDEPENDS_${PN} += " mraa mraa-java"
```

Comment:
There is no standard "java" distro feature, and thus meta-intel-iot-middleware cannot depend on it.

We recently introduced one in Ostro (https://github.com/ostroproject/meta-ostro/pull/216) and would have to adapt that to this change.

But I have a different comment: wouldn't it be better to split out the different language bindings into separate packages (like upm-java, upm-python, etc.)? Then even for the case where the distro has Java support enabled (like regular Ostro OS), images can still be created where "normal" upm gets installed without pulling in Java.